src/utils/number.util.ts

```diff
@@ -2,10 +2,20 @@ import { BigNumber } from 'bignumber.js';
 import { isNaN } from 'lodash-es';
 
 export const formatAssetAmount = (amount: BigNumber, decimalPlace = 6) => {
+  const MIN_DISPLAYED_AMOUNT = new BigNumber(0.000001);
+
   if (isNaN(amount.toNumber())) {
     return '';
   }
 
+  if (amount.isLessThan(MIN_DISPLAYED_AMOUNT) && !amount.isZero()) {
+    if (decimalPlace === 6) {
+      return '<0.000001';
+    } else {
+      return '0.01';
```

Comment:
how would `0.0001` value will be displayed when `decimalPlace = 2`?

Reply:
≈ 0.01$